lib/food_order.rb

```diff
@@ -0,0 +1,30 @@
+class FoodOrder
+  attr_accessor :basket, :takeaway_menu, :total_order
```

Comment:
It would be easy to build inheritance in your code. For example, we can have different types of menus with the menu class